pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/question/PicklistQuestion.java

```diff
@@ -155,6 +156,11 @@ public void registerTemplateIds(Consumer<Long> registry) {
 
         for (PicklistOption option : picklistOptions) {
             option.registerTemplateIds(registry);
+            if (CollectionUtils.isNotEmpty(option.getNestedPicklistOptions())) {
+                for (PicklistOption suboption : option.getNestedPicklistOptions()) {
+                    suboption.registerTemplateIds(registry);
```

Comment:
I feel like this should be done in the `PicklistOption` class instead, i.e., the option should register and apply templates to any nested options it contains.